### ISSUE TITLE
Messaging: unable to set distinct for conversation

### ISSUE THREAD

**Issue by ruscoder** [voximplant/react-native-voximplant#61]
I've found a problem with `distinct` flag when I create a new conversation. I've run the tests and got this error:
```
expected Conversation {
  createdAt: 1548669425,
  uuid: '***',
  lastRead: 0,
  participants: Array [
    Object {
      canWrite: true,
      canManageParticipants: true,
      userId: '****'
    },
    Object {
      canWrite: true,
      canManageParticipants: false,
      userId: '***'
    },
    Object {
      canWrite: true,
      canManageParticipants: true,
      userId: '***'
    }
  ],
  isUber: false,
  lastSeq: 1,
  publicJoin: true,
  distinct: false,
  lastUpdate: 1548669425,
  customData: Object {},
  title: 'Test conversation 1'
} to have property distinct of true (got false)
```

**Comment by YuliaGrigorieva** [comment]
Hello!

Recently we have updated our Messaging API and deprecated 'distinct' conversation. Now it is not possible to create a 'distinct' conversation, however the attempt to create a distinct conversation will not produce an error event, but create a new 'non-distinct' conversation. 

This is the reason why the test fails. 

In the new version on Messaging API we have introduced 'direct' conversations. Direct conversation is a conversation between 2 users only and is unique for these 2 users. Moreover direct conversation can't be uber or public. You can find more information in our Android SDK documentation: https://voximplant.com/docs/references/androidsdk/iconversation#isdirect

We are working on messaging branch update to add all new features of new Messaging API and promise support.

Best regards,
Yulia Grigorieva

**Comment by YuliaGrigorieva** [comment]
Hello!

Today we have released Voximplant React Native SDK 1.6.0 with Messaging API.

'messaging' branch will be completely removed from the repo on May 6th, 2019.

Best regards,
Yulia Grigorieva
